srcpkgs/66/template

```diff
@@ -0,0 +1,18 @@
+# Template file for '66'
+pkgname=66
+version=v0.1.0.0
```

Comment:
please don't use the `v` in the version

Reply:
The upstream version has the v ... Ι can change it if that presents any problem.

Reply:
i'd rather we don't hold the `v`

Reply:
OK, thanx :)